android/app/src/main/java/com/yagubogu/presentation/livetalk/LivetalkViewModel.kt

```diff
@@ -25,10 +25,16 @@ class LivetalkViewModel(
             val gamesResult: Result<List<LivetalkStadiumItem>> = gameRepository.getGames(date)
             gamesResult
                 .onSuccess { livetalkStadiumItems: List<LivetalkStadiumItem> ->
-                    _livetalkStadiumItems.value = livetalkStadiumItems
+                    _livetalkStadiumItems.value = sortStadiumsByVerification(livetalkStadiumItems)
                 }.onFailure { exception: Throwable ->
                     Timber.w(exception, "API 호출 실패")
                 }
         }
     }
+
+    private fun sortStadiumsByVerification(livetalkStadiumItems: List<LivetalkStadiumItem>): List<LivetalkStadiumItem> {
+        val verifiedStadiumItems = livetalkStadiumItems.filter { it.isVerified }
+        val unverifiedStadiumItems = livetalkStadiumItems.filter { !it.isVerified }
+        return verifiedStadiumItems + unverifiedStadiumItems
+    }
```

Comment:
이렇게 하면 리스트를 두번 순회할 필요가 없을 거 같아요!

```suggestion
    private fun sortStadiumsByVerification(livetalkStadiumItems: List<LivetalkStadiumItem>): List<LivetalkStadiumItem> =
        livetalkStadiumItems.sortedByDescending { it.isVerified }
```